rotkehlchen/exchanges/coinbase.py

```diff
@@ -162,7 +182,11 @@ def build_jwt(self, uri: str) -> str:
             RemoteError: If there is an error during the JWT token generation process.
         """
         try:
-            private_key = serialization.load_pem_private_key(self.secret, password=None)
+            if self.key_type == CoinbaseKeyType.ECDSA:
+                private_key = serialization.load_pem_private_key(self.secret, password=None)
+            else:  # can only be ED25519
+                private_key = ed25519.Ed25519PrivateKey.from_private_bytes(base64.b64decode(self.secret)[:32])  # noqa: E501
```

Comment:
can either of these raise an exception? And if yes is it handled anywhere?

Reply:
yeah, it's handled below with the `ValueError`